README.md

```diff
@@ -21,21 +25,24 @@ import { RDKitProvider } from '@iktos/rdkit-provider';
-| cache  | ```RDKitProviderCacheOptions = { enableJsMolCaching?: boolean; maxJsMolsCached?: number; }```  | enables ```JSMol``` caching for better performance | optional  |
-| preferCoordgen  | ```boolean```  | will be passed to [@rdkit/rdkitjs prefer_coordgen](https://docs.rdkitjs.com/interfaces/RDKitModule.html#prefer_coordgen.prefer_coordgen-1) to use Schrodinger’s open-source Coordgen library to generate 2D coordinates of molecules | optional  |  
-| removeHs  | ```boolean```  |  toggles removing hydrogens molecules. Defaults to true | optional  |  
-| ```initialRdkitInstance```  | ```RDKitModule``` from ```@rdkit/rdkitjs``` | pass an instance of to expose by the context `RDKitModule`, if not passed rdkit-provider creates one for you  | optional  |
+| prop                   | type                                                                                      | functionality                                                                                                                                                                                                                        | required/optional |
+| ---------------------- | ----------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------- |
+| cache                  | `RDKitProviderCacheOptions = { enableJsMolCaching?: boolean; maxJsMolsCached?: number; }` | enables `JSMol` caching for better performance                                                                                                                                                                                       | optional          |
+| preferCoordgen         | `boolean`                                                                                 | will be passed to [@rdkit/rdkitjs prefer_coordgen](https://docs.rdkitjs.com/interfaces/RDKitModule.html#prefer_coordgen.prefer_coordgen-1) to use Schrodinger’s open-source Coordgen library to generate 2D coordinates of molecules | optional          |
+| removeHs               | `boolean`                                                                                 | toggles removing hydrogens molecules. Defaults to true                                                                                                                                                                               | optional          |
+| `initialRdkitInstance` | `RDKitModule` from `@rdkit/rdkitjs`                                                       | pass an instance of to expose by the context `RDKitModule`, if not passed rdkit-provider creates one for you                                                                                                                         | optional          |
```

Comment:
I think`initialRdkitInstance` is no longer relevant

Reply:
Should be `initialWorkerInstance`, not rdkit module.
Changed it.
I added this props so we can pass an rdkit worker instance. It the same problem with previous version and rdkit module, I need a way to call the rdkit worker  outside react context.